lib/std/os/linux/tls.zig

```diff
@@ -168,6 +168,11 @@ pub fn setThreadPointer(addr: usize) void {
         .powerpc, .powerpc64, .powerpc64le => {
             asm volatile (
                 \\ mr 13, %[addr]
+            );
```

Comment:
this is removing the ppc clobbers/inputs/outputs

Reply:
Ouch, sorry. Seems that my rebase accidentally removed it.
Anyway, I've added it back.